src/tablecloth/api/join_separate.clj

```diff
@@ -116,3 +131,69 @@
      (if (grouped? ds)       
        (process-group-data ds #(process-separate-columns % column target-columns replace-missing separator-fn drop-column?) parallel?)
        (process-separate-columns ds column target-columns replace-missing separator-fn drop-column?)))))
+
+
+
+
+(defn- prefix [prefix-name value]
+   (let [with-prefix (str (name prefix-name) "-" value)]
```

Comment:
`->str` is from `tablecloth.api.utils` and checks for `Named` type. So, here, instead of `name` we should use `->str`. I think this is the last thing.